src/js/apps/patients/patient/dashboard/dashboard_app.js

```diff
@@ -50,7 +50,6 @@ export default App.extend({
   },
   subscribe() {
     const filters = {
-      states: this.states,
```

Comment:
On the pt dashboard, this currently stops us from receiving `Done` state changes on actions/flows added via web sockets. Because in that scenario the ws filters excludes done states and the action/flow isn't in the ws subscription resources list.

There are steps to reproduce it on the shortcut story.

So removing this from the ws filters means we'll receive ws notifications for flows/actions with any state. Which makes sense to me. The UI list only shows non-done state actions/flows, but we want to get ws notifications for all states.

But maybe i'm missing something. I can't remember why we added this ws filter in the first place.